Cargo.toml

```diff
@@ -11,10 +11,3 @@ members = [
   "utils",
 ]
 
-[patch."https://github.com/EspressoSystems/hotshot"]
```

Comment:
Was this supposed to go away?

Reply:
Oh I see - we're still developing off of that branch, I assumed we had merged to HotShot main